File: extras/read_excel.js
const xlsx = require('xlsx');

try {
  const filePath = "c:\\Users\\exbinario\\Documents\\desarrollo\\antigravity\\proyectoDixon\\extras\\IAD 158 DataHall Altas Bajas.xlsx";
  const workbook = xlsx.readFile(filePath);
  const sheetName = workbook.SheetNames[0];
  const sheet = workbook.Sheets[sheetName];
  const data = xlsx.utils.sheet_to_json(sheet, { header: 1 });
  
  for (let i = 15; i < Math.min(40, data.length); i++) {
    console.log(`Row ${i + 1}:`, JSON.stringify(data[i]));
  }
} catch (e) {
  console.error("Error reading file:", e.message);
}

File: extras/read_excel.py
import pandas as pd
import json

file_path = "c:\\Users\\exbinario\\Documents\\desarrollo\\antigravity\\proyectoDixon\\extras\\IAD 158 DataHall Altas Bajas.xlsx"

try:
    xl = pd.ExcelFile(file_path)
    output = {"sheet_names": xl.sheet_names, "sheets": {}}
    for sheet in xl.sheet_names:
        df = xl.parse(sheet)
        output["sheets"][sheet] = {
            "columns": df.columns.tolist(),
            "sample_row": df.head(1).to_dict(orient="records") if not df.empty else []
        }
    print(json.dumps(output, indent=2, default=str))
except Exception as e:
    print(f"Error: {e}")
